modules/server/src/startProject.js

```diff
@@ -58,6 +127,24 @@ export const createProjectEndpoint = async ({
 
   await initializeSets({ es });
 
+  return { schema, mockSchema };
+};
+
+export const createProjectEndpoint = async ({
+  es,
+  id,
+  graphqlOptions = {},
+  enableAdmin,
+  typesWithMappings,
+}) => {
+  const { schema, mockSchema } = createProjectSchema({
```

Comment:
@hlminh2000 This is the function that I've exported for argo gateway to use. Its literally everything before the apollo-router is created.